extensions/amp-ad/0.1/amp-ad-ui.js

```diff
@@ -80,11 +87,12 @@ export class AmpAdUIHandler {
     }
 
     // Apply default fallback div when there's no default one
-    const holder = createElementWithAttributes(document, 'div', {
-      'fallback': '',
-    });
-    holder.classList.add('amp-ad-default-fallback');
-    this.baseInstance_.element.appendChild(holder);
+    const fallback = this.doc_.createElement('div');
+    fallback.setAttribute('fallback', '');
+    fallback.classList.add('amp-ad-default-display');
+    fallback./*OK*/innerHTML = HOLDER_HTML;
+
+    this.baseInstance_.element.appendChild(fallback);
```

Comment:
I don't think it will work, `baseInstance_.element` may contain placeholder. overwriting innerHTML will also overwrite the placeholder info